### PR TITLE
Updates from FRC-2022 season repo

### PR DESCRIPTION
**Pull request by coderkevin**
This contains a number of updates from the FRC 2022 season where
this repo was copied into.

### CODE REVIEW

File: src/main/java/com/team2357/lib/util/RobotMath.java [hunk 1]
```diff
@@ -4,16 +4,17 @@
...
 	 * 
 	 * To start, visualize the center point of the robot with a horizontal line, and
-	 * another line rotated by the degrees desired to turn (counter-clockwise in this case)
+	 * another line rotated by the degrees desired to turn (counter-clockwise in
+	 * this case)
 	 * Using 'w' for where the wheels are now, and 'W' for where we want them to be,
 	 * we can draw two supplementary angles towards the arc of the wheels.
 	 * 
-	 *                 /W\
+	*                 /W\
```
Comment (Tyson453) [inline src/main/java/com/team2357/lib/util/RobotMath.java:17]: You accidentally got rid of a space.

File: src/main/java/com/team2357/lib/subsystems/drive/FalconDriveSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+     */
+    public void zeroHeading() {
+        m_gyro.setYaw(0);
+        m_gyro.setAccumZAngle(0);
+    }
+
+    /**
+     * Returns the heading of the robot.
+     *
+     * @return the robot's heading in degrees, from 180 to 180
```
Comment (Tyson453) [inline src/main/java/com/team2357/lib/subsystems/drive/FalconDriveSubsystem.java:358]:
```suggestion
     * @return the robot's heading in degrees, from -180 to 180
```

File: src/main/java/com/team2357/lib/commands/LimelightTargetingPipelineCommand.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * Changes the camera pipeline (Which camera stream is being shown).
+ * 
+ * @category Camera
+ */
+public class LimelightTargetingPipelineCommand extends CommandLoggerBase {
+    private boolean m_wait;
+
+    /**
+     */
```
Comment (Tyson453) [inline src/main/java/com/team2357/lib/commands/LimelightTargetingPipelineCommand.java:14]: Empty doc string.

File: src/main/java/com/team2357/lib/commands/LimelightHumanPipelineCommand.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+/**
+ * Changes the camera pipeline (Which camera stream is being shown).
+ * 
+ * @category Camera
+ */
+public class LimelightHumanPipelineCommand extends CommandLoggerBase {
+    private boolean m_wait;
+
+    /**
+     */
```
Comment (Tyson453) [inline src/main/java/com/team2357/lib/commands/LimelightHumanPipelineCommand.java:14]: Empty doc string.

File: src/main/java/com/team2357/lib/subsystems/LimelightSubsystem.java [hunk 5]
```diff
@@ -127,44 +61,52 @@ public static class Configuration {
...
+   */
+  public LimelightSubsystem() {
+    instance = this;
+  }
+
   public void setConfiguration(Configuration configuration) {
     m_Configuration = configuration;
+
+  //  setHumanPipelineActive();
+  setTargetingPipelineActive();
```
Comment (Tyson453) [inline src/main/java/com/team2357/lib/subsystems/LimelightSubsystem.java:79]: Missing indentation.

**Comment by coderkevin** [comment]
Thanks for the review @Tyson453 I think it's good to make adjustments to this code before we import it from last year's season.

**Review by NolanCampbell2357 — APPROVED** [review]

**Pull request merged**